### PR TITLE
edukai: init at 4.0/edusong: init at 1.0/eduli: init at 3.0

### PR DESCRIPTION
**Pull request by ShamrockLee**
###### Motivation for this change
These fonts are distributed by the Ministry of Education, Republic of China (Taiwan) under CC BY-ND 3.0.

The MOE Standard Kai Font (教育部標準楷書字形檔) and the MOE Standard Song Font (教育部標準宋體字形檔) follow the standard ("國字標準字體研訂原則") of the Ministry of Education in Taiwan. They would be useful in test sheets, especially when the commercial DFKai-SB (標楷體) is not available.

The MOE Li Font (教育部隸書字形檔) is a piece of font of [clerical script](https://en.wikipedia.org/wiki/Clerical_script) (an archaic style of Chinese calligraphy). If merged, it would probably become the first clerical font in nixpkgs.

I join them into one PR to avoid conflicts in `all-packages.nix`

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`) (No execution files)
- [X] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date (No document change needed)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Infinisil** [comment]
Trying to connect to a homepage of the packages I get

> Secure Connection Failed
>
> An error occurred during a connection to language.moe.gov.tw. Cannot communicate securely with peer: no common encryption algorithm(s).
>
> Error code: SSL_ERROR_NO_CYPHER_OVERLAP

So I propose to use HTTP instead of HTTPS for these links instead (both for the homepage and the download URL)

**Review by infinisil — CHANGES_REQUESTED** [review]

File: pkgs/data/fonts/edukai/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/
+    for font in ./*.ttf; do
+      chmod 644 $font
+      mv $font $out/share/fonts/
+    done
+  '';
```
[the remark this answers is not in the document]
Comment (ShamrockLee) [reply pkgs/data/fonts/edukai/default.nix:27]: Thanks a lot!
I added that line as `unzip` wasn't available by default when I wrote this.
I'll change them.

[the remark this answers is not in the document]
Comment (ShamrockLee) [reply pkgs/data/fonts/edukai/default.nix]: I have them renewed.
/status awaiting_reviewer

**Comment by ShamrockLee** [comment]
I have them simplified.
/status awaiting_reviewer

**Comment by Infinisil** [comment]
See also https://github.com/NixOS/nixpkgs/pull/93232#issuecomment-666478905

**Comment by ShamrockLee** [comment]
> See also [#93232 (comment)](https://github.com/NixOS/nixpkgs/pull/93232#issuecomment-666478905)

Oops! Thanks for reminding.

**Review by infinisil — APPROVED** [review]
Looking good!

**Pull request merged**